File: 0001 Code Chef/0003 Good Turn/implementation/0004 java/GoodTurn.java
import java.util.Scanner;


public class GoodTurn{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        String[] stringArray = stringInput.split(" ");
        int x = Integer.parseInt(stringArray[0]);
        int y = Integer.parseInt(stringArray[1]);
        int sumof = x + y;
        if(sumof > 6){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}

File: 0001 Code Chef/0003 Good Turn/implementation/0002 c++/good_turn.cpp
#include <iostream>

int main(){
    int x,y,sumof;
    std::cin >> x >> y;
    sumof = x + y;
    if(sumof > 6){
        std::cout << "YES" << std::endl;
    }else{
        std::cout << "NO" << std::endl;
    }
    return 0;
}